test/cpp/jit/test_inliner.cpp

```diff
@@ -39,26 +39,14 @@ struct InlinerGuard {
 
 void testInliner() {
   {
-    // Test that the recursive inlining works
     // disable automatic inlining so we can test it manually
     InlinerGuard guard(/*shouldInline=*/false);
 
     CompilationUnit cu(testSource);
     auto& fn = cu.get_function("foo3");
 
     auto g = fn.graph();
-    Inline(*g, /*recurse=*/true);
-    FileCheck().check_count("prim::Print", 3)->run(*g);
-  }
-  {
-    // disable automatic inlining so we can test it manually
-    InlinerGuard guard(/*shouldInline=*/false);
-
-    CompilationUnit cu(testSource);
-    auto& fn = cu.get_function("foo3");
-
-    auto g = fn.graph();
-    Inline(*g, /*recurse=*/false);
+    Inline(*g);
     FileCheck()
         .check("three")
         ->check("two")
```

Comment:
so `foo2` is inlined and `foo1` is not inlined right? then should you `check_not` one as well?

Reply:
oh, I saw check_count("prim::CallFunction", 1), but I think still better to add a "check_not" for "one" since that is what you are checking for previously

Reply:
We check the number of  `CallFunction` entries, if we inline everything it would be 0. What you suggest is another possible way to verify the same, but I don't think there is a reason to modify the existing test.

Reply:
This test has been checking non-recursive inlining before and I just kept it as-is :) I just removed the test for recursive inlining.

Reply:
Yeah, I saw this test is changed to check recursive inlining in later PRs.